package.json

```diff
@@ -25,7 +25,8 @@
     "publish-canary": "lerna version prerelease --preid canary --force-publish && release --pre",
     "publish-stable": "lerna version --force-publish",
     "lint-staged": "lint-staged",
-    "next": "./packages/next/dist/bin/next"
+    "next": "./packages/next/dist/bin/next",
+    "next-win": "node packages\\next\\dist\\bin\\next"
```

Comment:
Hmm what if you just try `"next": "node packages/next/dist/bin/next",`? From what I know, Windows CLI works with forward slashes in paths, it just does not understand [shebang](https://github.com/zeit/next.js/blob/63886b36a25dd371e6e6260647c7af53dd13fced/packages/next/bin/next.ts#L1).

Reply:
Prefixing it with `node` and using forward slashes seems to be the ideal way to handle this